packages/cli/src/commands/typescript/generate/generate.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2023 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { CommandModule } from "yargs";
+import type { TypescriptGenerateArgs } from "./TypescriptGenerateArgs.js";
+
+export const command: CommandModule<
+  {},
+  TypescriptGenerateArgs
+> = {
+  command: "generate",
+  describe: "Generate typescript from ontology",
+  builder: (argv) => {
+    return argv
+      .options({
+        "outDir": {
+          type: "string",
+          description: "Where to place the generated files",
+          demandOption: true,
+        },
+        ontologyPath: {
+          description: "Causes the site to no longer be deployed",
+          type: "string",
+          conflicts: "siteVersion",
+          demandOption: true,
+        },
+      }).group(
+        ["outDir", "ontologyPath"],
+        "Version To Deploy (requires one of)",
```

Comment:
copypaste description? also the ontologyPath `description` and `conflicts` entries

Reply:
Oops. thank you